generator/template.go

```diff
@@ -98,10 +98,114 @@ const fetchTmpl = `
 * This file is a generated Typescript file for GRPC Gateway, DO NOT MODIFY
 */
 
+// Base64 encoding table
```

Comment:
would be good to have a one line comment stating where these method comes from .